### PR TITLE
Include Greenhouse token in prospectus build (for careers page).

### PR DESCRIPTION
**Pull request by dianakhuang**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.

### CODE REVIEW

File: playbooks/roles/prospectus/templates/.env.environment.j2 [hunk 1]
```diff
@@ -5,3 +5,5 @@ OAUTH_SECRET={{ PROSPECTUS_OAUTH_SECRET }}
 
 NEW_RELIC_LICENSE_KEY={{ PROSPECTUS_NEW_RELIC_LICENSE_KEY }}
 NEW_RELIC_APP_ID={{ PROSPECTUS_NEW_RELIC_APP_ID }}
```
Comment (coryleeio) [inline playbooks/roles/prospectus/templates/.env.environment.j2:7]: You will also need to set the values here:
https://github.com/edx/configuration/blob/master/playbooks/roles/prospectus/defaults/main.yml#L15-L17

**Review by coryleeio — CHANGES_REQUESTED** [review]

File: playbooks/roles/prospectus/templates/.env.environment.j2 (hunk 1 above)
Comment (coryleeio) [reply playbooks/roles/prospectus/templates/.env.environment.j2:7]: @dianakhuang  If you lastpass share me the key I will add it for you to secure

Comment (dianakhuang) [reply playbooks/roles/prospectus/templates/.env.environment.j2:7]: @AlasdairSwan can you get DevOps the API key?

File: playbooks/roles/prospectus/defaults/main.yml [hunk 2]
```diff
@@ -19,6 +19,8 @@ PROSPECTUS_OAUTH_SECRET: !!null
 PROSPECTUS_NEW_RELIC_LICENSE_KEY: 'fake-key'
 PROSPECTUS_NEW_RELIC_APP_ID: 'fake-id'
 
+GATSBY_GREENHOUSE_API_KEY: 'fake-key'
```
Comment (coryleeio) [inline playbooks/roles/prospectus/defaults/main.yml]: this should be PROSPECTUS_NEW_RELIC_APP_ID
not GATSBY_GREENHOUSE_API_KEY

**Review by coryleeio — CHANGES_REQUESTED** [review]

**Comment by coryleeio** [comment]
@AlasdairSwan @dianakhuang Same key for stage vs prod?

**Comment by AlasdairSwan** [comment]
@coryleeio yes please.

File: playbooks/roles/prospectus/defaults/main.yml (hunk 2 above)
Comment (dianakhuang) [reply playbooks/roles/prospectus/defaults/main.yml]: I believe I did screw this up, but I fixed it to be `GATSBY_GREENHOUSE_API_TOKEN` instead.

File: playbooks/roles/prospectus/defaults/main.yml [hunk 3]
```diff
@@ -19,6 +19,8 @@ PROSPECTUS_OAUTH_SECRET: !!null
 PROSPECTUS_NEW_RELIC_LICENSE_KEY: 'fake-key'
 PROSPECTUS_NEW_RELIC_APP_ID: 'fake-id'
 
+GATSBY_GREENHOUSE_API_TOKEN: 'fake-key'
```
Comment (coryleeio) [inline playbooks/roles/prospectus/defaults/main.yml]:
```suggestion
PROSPECTUS_GREENHOUSE_API_TOKEN: 'fake-key'
```

**Review by coryleeio — CHANGES_REQUESTED** [review]

File: playbooks/roles/prospectus/defaults/main.yml (hunk 3 above)
Comment (coryleeio) [reply playbooks/roles/prospectus/defaults/main.yml]: This results in the same behavior and is consistent with the application specific values above it.

File: playbooks/roles/prospectus/templates/.env.environment.j2 (hunk 1 above)
Comment (coryleeio) [reply playbooks/roles/prospectus/templates/.env.environment.j2:7]: I have the key

File: playbooks/roles/prospectus/defaults/main.yml (hunk 3 above)
Comment (dianakhuang) [reply playbooks/roles/prospectus/defaults/main.yml]: Unfortunately, some of these values have to start with `GATSBY` for uh, gatsby-specific reasons. We can switch the key to `GATSBY_PROSPECTUS_GREENHOUSE_API_TOKEN` if absolutely necessary, but I'm not a big fan of making it that long.

**Review by coryleeio — APPROVED** [review]

**Pull request merged**